bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -863,6 +869,16 @@ def convert_transformation_from_BD_to_magnum(self, bd_transformation_dict: dict)
         mn_transformation = mn.Matrix4.from_(rotation_matrix, translation)
         return mn_transformation
 
+    def get_vision_tform_body_TransformStamped(self, frame_tree_snapshot):
```

Comment:
this will get refactored when function names are changed

Reply:
addressed